firestore.rules

```diff
@@ -20,20 +20,76 @@ service cloud.firestore {
       return collection == "user";
     }
 
-    match /users/{userId} {
+    match /users/{id} {
```

Comment:
Is there a specific reason for changing it to id? If not then I prefer the more verbose name.